libs/components/theme/src/lib/styles/_switch.scss

```diff
@@ -6,6 +6,10 @@
   display: inline-flex;
   position: relative;
 
+  &:has(~ .sky-control-help-container) .sky-switch-label {
+    margin-right: 0;
```

Comment:
This is to allow the use of the help container with switch labels without breaking existing spacing.